Mixpanel/MPNetwork.m

```diff
@@ -278,7 +278,7 @@ + (BOOL)parseHTTPFailure:(NSHTTPURLResponse *)response withError:(NSError *)erro
 }
 
 - (void)updateNetworkActivityIndicator:(BOOL)enabled {
-#if !MIXPANEL_LIMITED_SUPPORT
+#if !(defined(MIXPANEL_APP_EXTENSION) || defined(MIXPANEL_TVOS_EXTENSION) || defined(MIXPANEL_WATCH_EXTENSION))
```

Comment:
i thought about it but it wasn't going to be used elsewhere. i did the same thing in a couple other places where there wouldn't be a macro that could reasonably be reused. happy to change it if you want though